app/models/weet.js

```diff
@@ -0,0 +1,30 @@
+module.exports = (sequelize, DataTypes) => {
+  const Weet = sequelize.define(
+    'Weet',
+    {
+      userId: {
+        type: DataTypes.STRING,
+        allowNull: false
+      },
+      author: {
+        type: DataTypes.STRING,
+        allowNull: false
+      },
+      quote: {
+        type: DataTypes.STRING,
+        allowNull: false
+      }
+    },
+    {
+      timestamps: false,
+      underscored: true,
+      tableName: 'weets'
+    }
+  );
+
+  Weet.associate = models => {
+    Weet.belongsTo(models.User, { foreignKey: 'userId' });
```

Comment:
Great, but add the `hasMany()` association to the User model as well since this is a one-to-many relation.